notebooks/Mignardi_Pipeline.py

```diff
@@ -128,7 +128,7 @@
 disk_size_mask = None
 min_dist = 57
 
-seg = WatershedSegmenter(s.aux_dict['dapi'], s.aux_dict['stain'])
+seg = WatershedSegmenter(s.aux_dict['nuclei'], s.aux_dict['stain'])
```

Comment:
I think this change also needs to happen in the `ISS_Pipeline_-_Breast_-_1_FOV.[py|ipynb]` files. Related: We should write a CI function to run our `.py` encoded notebooks (and ideally the notebooks themselves), as discussed yesterday.